2018-07-31 14:38:11 INFO  /home/ktian/kundajelab/tfnet/scripts/run_pipeline.py -1
Traceback (most recent call last):
  File "/home/ktian/kundajelab/tfnet/scripts/run_pipeline.py", line 49, in <module>
    os.chdir("model_files")
OSError: [Errno 2] No such file or directory: 'model_files'
